UEFITool/ffsfinder.cpp

```diff
@@ -29,7 +29,12 @@ USTATUS FfsFinder::findHexPattern(const UModelIndex & index, const UByteArray &
 
     bool hasChildren = (model->rowCount(index) > 0);
     for (int i = 0; i < model->rowCount(index); i++) {
+#ifdef _USE_DEPRECATED
```

Comment:
Why cannot we use non-deprecated interfaces exclusively? Does Qt 5.6 not support them? Who does define `_USE_DEPRECATED`?

Reply:
I'll do a version check.
Fixing in a minute